apps/web/src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx

```diff
@@ -228,7 +228,12 @@ const IfoCardTokens: React.FC<React.PropsWithChildren<IfoCardTokensProps>> = ({
       publicIfoData.status !== 'finished' &&
       hasProfile
     ) {
-      message = <ICakeTips ifoId={ifo.id} ifoChainId={ifo.chainId} ifoAddress={ifo.address} />
+      // If Cross-Chain IFO
+      // if (ifo.chainId !== ChainId.BSC) {
+      message = <CrossChainVeCakeTips ifoChainId={ifo.chainId} />
+      // }
+      // Phase this out later, as it applies at the same time
+      // else message = <ICakeTips ifoId={ifo.id} ifoChainId={ifo.chainId} ifoAddress={ifo.address} />
```

Comment:
can we remove these comments or may they be needed in future?

Reply:
I would prefer to keep it for now, as might need that line of code for BNB chain IFO in future just in case

Reply:
yeah i was kinda thinking that. just wanted to comment just incase. cool